exercises/G-conditionals-2/exercise-3.js

```diff
@@ -8,7 +8,13 @@
   - Otherwise the grade is "F"
 */
 
-function calculateGrade(mark) {}
+function calculateGrade(mark) {
+  if (mark >= 80) return "A";
+  else if (60 < mark < 80) return "B";
```

Comment:
Hey Deniz,

Hope you are well!
This code looks great. You could even simplify it further by changing it to ** else if mark >60 return "B"; **
The reason you wouldn't need to check if it is< 80 is that it has already checked on line 12 before passing down to the else statement. 

You can also make a similar change to line 14.

Hope that makes sense!

Nigel

Reply:
Hello Nigel,

I'm well, hope you are good too. You are right, I just edited my code, thank you so much.